src/components/KTable/KTable.vue

```diff
@@ -84,18 +84,17 @@
         :data-tableid="tableId"
       >
         <thead :class="{ 'is-scrolled': isScrolled }">
-          <tr :class="{ 'is-scrolled': isScrolled }">
+          <tr
+            ref="headerRow"
+            :class="{ 'is-scrolled': isScrolled }"
+          >
             <th
               v-for="(column, index) in tableHeaders"
               :key="`k-table-${tableId}-headers-${index}`"
-              :aria-sort="!disableSorting && column.key === sortColumnKey ? (sortColumnOrder === 'asc' ? 'ascending' : 'descending') : undefined"
-              class="k-table-headers"
               :class="{
-                'sortable': !disableSorting && !column.hideLabel && column.sortable,
-                'active-sort': !disableSorting && !column.hideLabel && column.sortable && column.key === sortColumnKey,
-                [sortColumnOrder]: !disableSorting && column.key === sortColumnKey && !column.hideLabel,
-                'is-scrolled': isScrolled
+                'resize-hover': currentFocusedColumn === column.key && resizeColumns,
```

Comment:
we track the currently focused column to show/hide the right border of the column